src/http.rs

```diff
@@ -0,0 +1,22 @@
+use curl;
+use oauth2::*;
+use app::App;
+
+pub fn github(app: &App, url: &str, auth: &Token)
+    -> Result<curl::http::Response, curl::ErrCode> {
+    println!("HTTP: {}", url);
+    app.handle()
+     .get(url)
+     .header("Accept", "application/vnd.github.v3+json")
+     .header("User-Agent", "hello!")
+     .auth_with(auth)
+     .exec()
+}
+
+pub fn token(token: String) -> Token {
```

Comment:
Could you add a comment here indicating that this is intended to pass to the function above? (e.g. the scopes/token_type are blank)